TLM/TLM/Manager/Impl/ExtCitizenInstanceManager.cs

```diff
@@ -1085,7 +1085,7 @@ > ItemClass.Service.Office
                 // NON-STOCK CODE END
                 if (res) {
                     if (logParkingAi) {
-                        Log._Debug(string.Format(
+                        Log._DebugFormat(
```

Comment:
Would it be worth adding a `Log._DebugFormatIf( switch, ... )`?

Reply:
For brevity, yes. It will make the code look shorter.
For performance, no. In Debug, when call to log is not eliminated, it will still have to build an array of args passed inside `_DebugFormatIf`, and only there it will be able to check the condition. Although that is cheap but still that is an allocation and a few pointer copies.